lib/provider/immoscout.js

```diff
@@ -76,8 +120,87 @@ async function getListings(url) {
         link: `${metaInformation.baseUrl}expose/${item.id}`,
         address: item.address?.line,
         image,
+        additionalpictures: extractedAdditionalPictures,
+        detailedinfo: null,
       };
     });
+
+  // Fetch detailed info for all items in parallel and attach results
+  await Promise.all(
```

Comment:
I was thinking about that and even playing around with this, but I strongly suggest _not_ to do this, because this will spawn way too many requests. It _might_ work for Immoscout, however it will not work for others. Immonet/Immowelt, all of which work the "classic" way will detect these parallel calls as bots, thus subsequent calls will fail.